### PR TITLE
Explicitly set line length for prettier in the prettier config

### PR DESCRIPTION
**Pull request by azangru**
Hopefully after this prettier will behave identically on our machines.

Reference: [prettier docs](https://prettier.io/docs/en/options.html#print-width)

### CODE REVIEW

File: src/ensembl/prettier.config.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 module.exports = {
   singleQuote: true,
   arrowParens: 'always',
-  trailingComma: "none"
+  trailingComma: "none",
```
Comment (imransl) [inline src/ensembl/prettier.config.js:4]: Use single quotes instead?

**Comment by imransl** [comment]
Strange that the CI/CD is breaking despite this change not seeming to be one to do so

File: src/ensembl/prettier.config.js (hunk 1 above)
Comment (azangru) [reply src/ensembl/prettier.config.js:4]: I only added a comma on this line...

Comment (azangru) [reply src/ensembl/prettier.config.js]: Updated

Comment (imransl) [reply src/ensembl/prettier.config.js]: Thanks. Yes I know it isn't related to PR but thought we'll just change it here itself 😄

**Review by imransl — APPROVED** [review]

**Pull request merged**